### PR TITLE
add flutter learning id

### PR DESCRIPTION
**Pull request by Dicky1**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by LuigiImVector** [comment]
The new category is not ordered alphabetically. Check [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)

**Comment by davorpa** [comment]
:writing_hand: It's a good practice to fill Pull Request fields. It helps a bit we can understand your contribution better :hugs:.

File: books/free-programming-books-id.md [hunk 1]
```diff
@@ -161,3 +162,8 @@
 ### Vue
 
 * [Dokumentasi Vue Bahasa Indonesia](https://github.com/vuejs-id/docs)
+
+
+#### Flutter
+
+* [Flutter Widget](https://belajarflutter.com/flutter-widget-elevatedbutton/)
```
Comment (davorpa) [inline books/free-programming-books-id.md:169]: It seems like a blog post / article... 🤔. I need some feedback about

File: books/free-programming-books-id.md [hunk 2]
```diff
@@ -21,6 +21,7 @@
   * [Angular](#angular)
   * [Deno](#deno)
 * [Vue](#vue)
+* [Flutter](#flutter)
```
Comment (davorpa) [inline books/free-programming-books-id.md:24]: Fix list alphabetize-lists lint rule moving between 

https://github.com/EbookFoundation/free-programming-books/blob/699d51c27291eff63d2a86a02e5fb45494d4bce9/books/free-programming-books-id.md#L7-L8

Do the same with related section 😉

**Review by davorpa — CHANGES_REQUESTED** [review]
It's a good moment to make some changes. When done, request for a review again using GitHub :arrows_clockwise: button.

Closes #5870

**Comment by davorpa** [comment]
Domain already listed

https://github.com/EbookFoundation/free-programming-books/blob/eee3067457bb0a2fc2ad0b46cf5dadc3a2b70302/books/free-programming-books-id.md#L62

**Comment by davorpa** [comment]
Duplicate of #6167

**Pull request closed without merging**